src/plugins/es_ui_shared/public/components/cron_editor/cron_daily.js

```diff
@@ -52,6 +53,9 @@ export const CronDaily = ({
           <EuiSelect
             options={hourOptions}
             value={hour}
+            label={i18n.translate('esUi.cronEditor.cronDaily.hourSelectLabel', {
```

Comment:
I think this needs to be `aria-label` for it to be read aloud by a screen-reader. I used [VoiceOver](https://webaim.org/articles/voiceover/) to test this, and it won't announce these values unless I change them to `aria-label`.

Reply:
Good catch! I changed this to `aria-label` and tested with voiceover.